lib/pandadoc.ex

```diff
@@ -31,10 +31,10 @@ defmodule Pandadoc do
   @type client() :: Tesla.Client.t()
   @type result() :: {:ok, map() | String.t(), Tesla.Env.t()} | {:error, map(), any}
 
-  @spec client(String.t()) :: client()
-  def client(api_key) do
+  @spec client(String.t(), String.t()) :: client()
+  def client(api_key, base_url \\ "https://api.pandadoc.com/public/v1") do
```

Comment:
@avinayak would maybe make this a constant `@prod_pandadoc_base_url "https://api.pandadoc.com/public/v1"`